sdk/src/Core/Amazon.Runtime/Internal/Transform/SimpleTypeUnmarshaller.cs

```diff
@@ -921,7 +921,10 @@ public List<I> Unmarshall(JsonUnmarshallerContext context)
         {
             context.Read(); // Read [ or null
             if (context.CurrentTokenType == JsonToken.Null)

```

Comment:
Since this is no longer just a simple one line return use brackets for this `if` block.